components/TimePeriod/index.js

```diff
@@ -1,14 +1,20 @@
+import DateIcon from "@/components/TimePeriod/DateIcon.svg";
+import { DateContainer } from "./TimePeriod.styled";
+
 export default function TimePeriod({ timePeriod }) {
   return (
-    <time>
-      {timePeriod.map(
-        (timePeriod) =>
-          `${
-            !timePeriod.endDate
-              ? `${timePeriod.startDate}`
-              : `${timePeriod.endDate} - ${timePeriod.startDate}`
-          }`
-      )}
-    </time>
+    <DateContainer>
+      <DateIcon />
+      <time>
+        {timePeriod.map(
+          (timePeriod) =>
+            `${
+              !timePeriod.endDate
+                ? `${timePeriod.startDate}`
+                : `${timePeriod.startDate} - ${timePeriod.endDate}`
+            }`
```

Comment:
Why all the dollaring?

Reply:
We will adjust the dollaring in line 13 as we don't need it here, but we would keep line 14 since we want to have startDate and endDate written in one line